cpp/example_code/cognito/README.md

```diff
@@ -0,0 +1,86 @@
+get access tokens.
+
+*Amazon Cognito provides authentication, authorization, and user management for your web and mobile apps. Your users can sign in directly with a user name and password, or through a third party such as Facebook, Amazon, Google or Apple.*
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Confirm a user](./getting_started_with_user_pools.cpp) (ConfirmSignUp)
+* [Delete a user](./getting_started_with_user_pools.cpp) (DeleteUser)
+* [Get a token to associate an MFA application with a user](./getting_started_with_user_pools.cpp) (AssociateSoftwareToken)
+* [Get information about a user](./getting_started_with_user_pools.cpp) (AdminGetUser)
+* [Resend a confirmation code](./getting_started_with_user_pools.cpp) (ResendConfirmationCode)
+* [Respond to SRP authentication challenges](./getting_started_with_user_pools.cpp) (RespondToAuthChallenge)
+* [Sign up a user](./getting_started_with_user_pools.cpp) (SignUp)
+* [Start authentication with a tracked device](./getting_started_with_user_pools.cpp) (InitiateAuth)
+* [Verify an MFA application with a user](./getting_started_with_user_pools.cpp) (VerifySoftwareToken)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+* [Sign up a user with a user pool that requires MFA](./getting_started_with_user_pools.cpp) (ConfirmSignUp, DeleteUser,
+  AssociateSoftwareToken, AdminGetUser, ResendConfirmationCode, RespondToAuthChallenge, SignUp, InitiateAuth, VerifySoftwareToken)
+
+## Run the examples
+
+### Prerequisites
+Before using the code examples, first complete the installation and setup steps
+for [Getting started](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html) in the AWS SDK for
+C++ Developer Guide.
+This section covers how to get and build the SDK, and how to build your own code by using the SDK with a
+sample Hello World-style application.
+
+Next, for information on code example structures and how to build and run the examples, see [Getting started with the AWS SDK for C++ code examples](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started-code-examples.html).
+
```

Comment:
Does (and should?) the Readme include information on how to set up the user pool using the CDK script?